src/FGame.cpp

```diff
@@ -125,8 +129,7 @@ void FGame::render()
 	// Render characters
 	for (auto &character : m_engine->getCharacters())
 	{
-		//TODO Store filename in a unordered_map from character ID (int) to string
-		auto sprite = loadSprite("images/business.png", character.getSize().x, character.getSize().y);
+		auto sprite = loadSprite(m_characterTypeMap[character.getType()], character.getSize().x, character.getSize().y);
```

Comment:
*GENIUS*.
We just make a similar map for each type.
I think we could/should do the same thing for the tiles on the level as well.